lib/src/clojure_lsp/refactor/transform.clj

```diff
@@ -821,20 +813,20 @@
 
 (defn ^:private create-function-for-alias
   [local-zloc ns-or-alias fn-name defn-edit uri db]
-  (let [filename (shared/uri->filename uri)
-        ;; TODO: shouldn't this also look for unaliased ns-usages?
+  (let [;; TODO: shouldn't this also look for unaliased ns-usages?
         ;; See https://github.com/clojure-lsp/clojure-lsp/issues/1023
-        ns-usage (q/find-namespace-usage-by-alias db filename (symbol ns-or-alias))
+        ns-usage (q/find-namespace-usage-by-alias db uri (symbol ns-or-alias))
         ns-definition (when ns-usage
                         (q/find-definition db ns-usage))
-        source-paths (settings/get db [:source-paths])
+        source-path (shared/uri->source-path uri (settings/get db [:source-paths]))
```

Comment:
Another place we still use filenames—to figure out what URI a namespace should correspond to.